docs/language_guide.md

```diff
@@ -522,37 +634,57 @@ struct Rectangle {
 ```
 
 ### Instances
-The `Rectangle` structure definition describes only what a `Rectangle` would look like and what functionality they contain. They don't describe specific rectangles. To do that you create an instance of that structure.
+
+The declaration of a struct only describes what types of variables it contains, what their initial values are, and what functions may be used to modify or access the struct data. To create concrete instances, each with individual data values, an instance has to be created, by calling the initialiser of a struct.
+
+```swift
+<struct-name>(<initialiser-parameter-values>)
+```
+
+Example:
 
 ```swift
 let someRectangle: Rectangle = Rectangle()
 ```
 
-When you create an instance, you initialize that structure with it's initial values - in this case a width and height of 0. This process can also be done manually using the `init` special declaration. You can access the properties of the current structure with the special keyword `self`
+When an instance is created, it is initialised with its initial values – in this case a width and heigth of `0`. This process can also be done manually using an [initialiser](#initialisers). You can access the properties of the current struct with the special keyword `self`.
+
+Example:
 
 ```swift
 struct Rectangle {
   // Same definition as above with:
-  public init(width: Int, height: Int){
+  public init(width: Int, height: Int) {
     self.width = width
     self.height = height
   }
 }
 ```
 
 ```swift
-let bigRectangle = Rectangle(400, 10000)
+let bigRectangle = Rectangle(width: 400, height: 10000)
 ```
 
-### Accessing Properties/Functions
-You can access properties/functions of an instance using _dot syntax_. In dot syntax, you write the property name immediately after the instance name, separated by a period (.), without any spaces.
+### Accessing properties/functions
 
+Properties/functions of a struct instance can be accessed using dot syntax. In dot syntax, the property name is written immediately after the instance name, separated by a period `.`:
```

Comment:
Functions in the context of an instance are called methods. So this should say `Properties and methods of a struct instance`

Reply:
There is also no other syntax other than dot syntax for accessing struct members, so I don't see a need for explicitly pointing out that it's dot syntax